ecl/hqlcpp/hqlres.cpp

```diff
@@ -313,13 +314,13 @@ void ResourceManager::addManifestsFromArchive(IPropertyTree *archive, ICodegenCo
         const char *xml = manifestContents;
         bool isSigned = false;
         // Check for signature
-        if (startsWith(xml, "-----BEGIN PGP SIGNED MESSAGE-----"))
+        if (queryCodeSigner().hasSignature(xml))
         {
             try
             {
-                OwnedHqlExpr sig = checkSignature(manifestContents.length(), manifestContents.str());
-                xml = stripSignature(strippedManifestContents, manifestContents).str();
-                isSigned = true;
+                StringBuffer signer, errmsg;
+                isSigned = queryCodeSigner().verifySignature(manifestContents, signer, errmsg);
```

Comment:
no longer throws?

Reply:
I've changed verifySignature to throw exceptions instead of return error messages in errmsg.